components/rhs_comment/rhs_comment.tsx

```diff
@@ -500,7 +500,7 @@ export default class RhsComment extends React.PureComponent<Props, State> {
         const postClass = classNames('post__body', {'post--edited': PostUtils.isEdited(this.props.post)});
 
         let fileAttachment: ReactNode = null;
-        if (post.file_ids && post.file_ids.length > 0) {
+        if ((post.file_ids && post.file_ids.length > 0) || (post.file_client_ids && post.file_client_ids.length > 0)) {
```

Comment:
this can also be simplified:
```suggestion
        if (post.file_ids?.length || post.file_client_ids?.length) {
```